labelbox/schema/bulk_import_request.py

```diff
@@ -633,13 +633,12 @@ def validate_subclasses(cls, value, field):
         #This is caused by the fact that we require these ids for top level classifications but not for subclasses
         results = []
         for row in value:
-            row.update({
-                'dataRow': {
+            results.append({
+                **row, 'dataRow': {
                     'id': 'child'.center(25, '_')
```

Comment:
I would maybe move `'child`.center(25, '_')` out of the loop, it makes it more clear that it's just a dummy fill in value.

Reply:
Good call!

Reply:
Fixed.